test/test_nn.py

```diff
@@ -20862,6 +20867,17 @@ def test_multihead_attn_in_proj_weight_none(self, device, dtype):
         key = torch.rand(4, 4, 2, dtype=dtype, device=device)
         mha(query, key, key)
 
+    @onlyCPU
```

Comment:
Why only CPU?

Reply:
Test is specifically targeted at addressing issue #81129 which only occurs on CPU